server/search.py

```diff
@@ -53,7 +53,7 @@ def _get_variants_in_range(self, chr, start, end, data_type):
 
     def _search_gene(self, query, data_type):
         c = self.conn[threading.get_ident()].cursor()
-        c.execute('SELECT * FROM genes WHERE gene_name = ? LIMIT 1', [query])
+        c.execute('SELECT * FROM genes WHERE gene_name = ? OR gene_name = ? LIMIT 1', [query, query.upper()])
```

Comment:
in the end best to uppercase all gene names when importing and then always uppercasing query. Now we'd miss mixed case names genes which in unlikely to be the case for us but better prepare for that also.